### PR TITLE
Yro/EDUCATOR-2701

### PR DESCRIPTION
**Pull request by yro**
### Description
Add a class method to blank out `ip_address` field for user on `ArticleRevision` model class.

### Testing
This lib appears to lack unit tests. This will be unit tested on the LMS side and manually before launch.

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review: @iloveagent57 

FYI: @edx/educator-neem 

### Post-review
- [ ] Rebase and squash commits

### CODE REVIEW

**Review by schenedx — APPROVED** [review]
👍

File: wiki/models/article.py [hunk 1]
```diff
@@ -290,7 +297,7 @@ def save(self, *args, **kwargs):
             not self.previous_revision and
             self.article and
             self.article.current_revision and
-            self.article.current_revision != self):
+                self.article.current_revision != self):
```
Comment (iloveagent57) [inline wiki/models/article.py]: Too much indentation?

File: wiki/models/article.py [hunk 2]
```diff
@@ -308,8 +315,18 @@ def save(self, *args, **kwargs):
             self.article.current_revision = self
             self.article.save()
 
+    @classmethod
+    def retire_user(cls, user):
+        """
+        Delete user record as a part of GDPR Phase II
```
Comment (iloveagent57) [inline wiki/models/article.py]: Updates a record, not deletes...

File: wiki/models/article.py [hunk 3]
```diff
@@ -308,8 +315,18 @@ def save(self, *args, **kwargs):
...
+        :param user (obj)
+        :return: bool
+        """
+
+        article_revisions = ArticleRevision.objects.filter(user=user).update(ip_address='')
```
Comment (iloveagent57) [inline wiki/models/article.py]: s/ArticleRevision/cls/

**Review by iloveagent57 — COMMENTED** [review]
Looks good, really minor points.  Thanks for cleaning up those pep8 violations.

File: wiki/models/article.py (hunk 2 above)
Comment (yro) [reply wiki/models/article.py]: @iloveagent57 Thank you. UGH.

File: wiki/models/article.py (hunk 3 above)
Comment (yro) [reply wiki/models/article.py]: 👍

File: wiki/models/article.py (hunk 1 above)
Comment (yro) [reply wiki/models/article.py]: What's the standard for this type of thing? I'll just do one line.

Comment (yro) [reply wiki/models/article.py]: Scratch that. Gonna just put it back 🤕

**Pull request merged**